.idea/php.xml

```diff
@@ -71,7 +73,7 @@
       </interpreter>
       <interpreter id="ba314a65-6074-40c9-8112-007ea3db0a5f" name="7.2 (test)" home="docker-compose://DATA" debugger_id="php.debugger.XDebug" debugger_extension="/usr/local/lib/php/extensions/no-debug-non-zts-20170718/xdebug.so">
         <configuration_options>
-          <configuration_option name="xdebug.remote_host" value="host.docker.internal" />
+          <configuration_option name="xdebug.remote_host" value="172.17.0.1" />
```

Comment:
AFAIK it should be `host.docker.internal`, which is mapped to whatever IP via the `docker-compose.yml`.

Reply:
`host.docker.internal` is Windows-specific, while AFAIK `172.17.0.1` is cross-platform.

Reply:
`host.docker.internal` is Windows specific with what Docker does automatically. But in `php-project` setup we simply map the configured IP to `hosts.docker.internal`, and so it will exist either way. If you can find  where `172.17.0.1` is being documented to be a cross-platform way to access the host, I can put it into `.env.example`, and it should work for everyone 👍

Reply:
Let's continue this discussion in a more appropriate channel.